file.go

```diff
@@ -75,11 +72,11 @@ func (e *FileError) Error() string {
 type File struct {
 	// ID is a client defined string used as a reference to this record
 	ID string `json:"id"`
-	// FileHeader is an ICL FileHeader
+	// FileHeader is a ileHeader
```

Comment:
"a FileHeader" ?

Reply:
Fixed and will be pushed to initial design and development